docs/integrations/optuna.rst

```diff
@@ -3,150 +3,214 @@
 Neptune-Optuna Integration
 ==========================
 
-This integration enables you to monitor |Optuna| hyperparameter optimization in Neptune.
+|Run on Colab|
 
-.. image:: ../_static/images/integrations/optuna_neptuneai.png
-   :target: ../_static/images/integrations/optuna_neptuneai.png
-   :alt: Optuna Neptune integration
+What will you get with this integration?
+----------------------------------------
 
-Requirements
-------------
-To use Neptune + Optuna integration you need to have installed is |neptune-client| and |neptune-contrib|.
+|optuna-tour|
 
-.. code-block:: bash
+|Optuna| is an open source hyperparameter optimization framework to automate hyperparameter search. With Neptune integration, you can:
 
-    pip install neptune-client neptune-contrib['monitoring']
+* see the experiment as it is running,
+* see charts of logged run scores,
+* log the parameters tried at every run,
+* log the best parameters after training,
+* explore interactive optuna visualizations like plot_contour, plot_slice, plot_parallel_coordinate, optimization_history,
+* save the study object.
+   
+.. note::
 
+    This integration is tested with ``optuna==2.3.0`` and current latest, ``neptune-client==0.4.125`` and current latest, and ``neptune-contrib==0.24.7`` and current latest.
 
-Initialize Neptune and create an experiment
--------------------------------------------
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`Quickstart <quickstart>` below. You can also skip the basics and take a look at the :ref:`advanced options <advanced-options>`.
 
-.. code-block:: python3
+If you want to try things out and focus only on the code you can either:
 
-    import neptune
+#. Open the Colab notebook (badge-link below) with quickstart code and run it as an anonymous user "`neptuner`" - zero setup, it just works,
+#. View quickstart code as a plain Python script on |script|.
 
-    neptune.init(api_token='ANONYMOUS',
-                 project_qualified_name='shared/showroom')
+.. _quickstart:
 
-    neptune.create_experiment('optuna-sweep')
+Quickstart
+----------
+This quickstart will show you how to:
 
-Create **NeptuneCallback**
---------------------------
-.. note:: You can pass the experiment object as the first argument if you want to do it explicitly.
+* Install the necessary neptune packages
+* Connect Neptune to your script and create the first experiment
```

Comment:
I think those feel very generic (almost as if it is for some other tutorial). 
Please adjust to mention that you will log your first optuna sweep or something like that.